File: WebApp/src/app/shared/models/document/csBookingNote.ts
import { SystemConstants } from "src/constants/system.const";

export class csBookingNote {
    id: string = SystemConstants.EMPTY_GUID;
    jobId: string = null;
    from: string = null;
    telFrom: string = null;
    to: string = null;
    telTo: string = null;
    revision: string = null;
    bookingNo: string = null;
    shipperId: string = null;
    shipperDescription: string = null;
    consigneeId: string = null;
    consigneeDescription: string = null;
    dateOfStuffing: string = null;
    closingTime: string = null;
    placeOfStuffing: string = null;
    contact: string = null;
    etd: string = null;
    eta: string = null;
    pol: string = null;
    pod: string = null;
    vessel: string = null;
    voy: string = null;
    paymentTerm: string = null;
    freightRate: string = null;
    placeOfDelivery: string = null;
    noOfContainer: string = null;
    commodity: string = null;
    specialRequest: string = null;
    gw: string = null;
    cbm: string = null;
    serviceRequired: string = null;
    otherTerms: string = null;
    hblNo: string = null;
    noOfBl: string = null;
    pickupAt: string = null;
    dropoffAt: string = null;
    note: string = null;
    //
    packageQty: number = null;

    bookingDate: string = null;
    userCreated: string = null;
    createdDate: string = null;
    userModified: string = null;
    datetimeModified: string = null;
    creatorName: string = null;
    modifiedName: string = null;

    constructor(object?: any) {
        const self = this;
        for (const key in object) {
            if (self.hasOwnProperty(key.toString())) {
                self[key] = object[key];
            }
        }
    }
}

File: WebAPI/eFMS.API.SystemWeb/eFMS.API.Documentation/eFMS.API.Documentation.DAL/Models/CsBookingNote.cs
using System;
using System.Collections.Generic;

namespace eFMS.API.Documentation.Service.Models
{
    public partial class CsBookingNote
    {
        public Guid Id { get; set; }
        public Guid? JobId { get; set; }
        public string From { get; set; }
        public string TelFrom { get; set; }
        public string To { get; set; }
        public string TelTo { get; set; }
        public string Revision { get; set; }
        public string BookingNo { get; set; }
        public DateTime? BookingDate { get; set; }
        public string ShipperId { get; set; }
        public string ShipperDescription { get; set; }
        public string ConsigneeId { get; set; }
        public string ConsigneeDescription { get; set; }
        public DateTime? DateOfStuffing { get; set; }
        public DateTime? ClosingTime { get; set; }
        public string PlaceOfStuffing { get; set; }
        public string Contact { get; set; }
        public DateTime? Etd { get; set; }
        public DateTime? Eta { get; set; }
        public Guid? Pol { get; set; }
        public Guid? Pod { get; set; }
        public string Vessel { get; set; }
        public string Voy { get; set; }
        public string PaymentTerm { get; set; }
        public string FreightRate { get; set; }
        public string PlaceOfDelivery { get; set; }
        public string NoOfContainer { get; set; }
        public string Commodity { get; set; }
        public string SpecialRequest { get; set; }
        public decimal? Gw { get; set; }
        public decimal? Cbm { get; set; }
        public string ServiceRequired { get; set; }
        public string OtherTerms { get; set; }
        public string HblNo { get; set; }
        public string NoOfBl { get; set; }
        public string PickupAt { get; set; }
        public string DropoffAt { get; set; }
        public string Note { get; set; }
        public string UserCreated { get; set; }
        public DateTime? CreatedDate { get; set; }
        public string UserModified { get; set; }
        public DateTime? DatetimeModified { get; set; }
        public decimal? PackageQty { get; set; }
    }
}
